src/main/lib/configuration.properties

```diff
@@ -9,6 +9,7 @@ otversion = 2.4.6
 default.cascade = merge
 temp-file-name-scheme = org.dita.dost.module.GenMapAndTopicListModule$DefaultTempFileScheme
 cli.color = true
+preprocessed.ditaval.name = ditaot.generated.ditaval
```

Comment:
IMO this doesn't have to be configurable and the name can be just a constant in Java code.